packages/tools/client-debugger/package.json

```diff
@@ -1,102 +1,102 @@
-    "format-and-build": "npm run format && npm run build",
-    "format-and-compile": "npm run format && npm run build:compile",
-    "lint": "npm run prettier && npm run eslint",
-    "lint:fix": "npm run prettier:fix && npm run eslint:fix",
-    "mocha": "mocha dist/**/*.test.js --unhandled-rejections=strict --exit",
-    "mocha:coverage": "nyc npm run mocha -- --reporter xunit --reporter-option output=nyc/junit-report.xml",
-    "prettier": "prettier --check .",
-    "prettier:fix": "prettier --write .",
-    "start:tinylicious": "tinylicious",
-    "test": "npm run test:mocha",
-    "test:coverage": "npm run test:mocha:coverage",
-    "test:mocha": "start-server-and-test start:tinylicious 7070 mocha",
-    "test:mocha:coverage": "start-server-and-test start:tinylicious 7070 mocha:coverage",
-    "tsc": "tsc --pretty"
-  },
-  "nyc": {
-    "all": true,
-    "cache-dir": "nyc/.cache",
-    "exclude": [
-      "src/test/**/*.ts",
-      "dist/test/**/*.js"
-    ],
-    "exclude-after-remap": false,
-    "include": [
-      "src/**/*.ts",
-      "dist/**/*.js"
-    ],
-    "report-dir": "nyc/report",
-    "reporter": [
-      "cobertura",
-      "html",
-      "text"
-    ],
-    "temp-directory": "nyc/.nyc_output"
-  },
-  "dependencies": {
-    "@fluidframework/common-definitions": "^0.20.1",
-    "@fluidframework/common-utils": "^1.0.0",
-    "@fluidframework/container-definitions": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/container-loader": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/core-interfaces": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/driver-definitions": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/protocol-definitions": "^1.1.0"
-  },
-  "devDependencies": {
-    "@fluidframework/build-common": "^1.1.0",
-    "@fluidframework/build-tools": "^0.6.0-109663",
-    "@fluidframework/counter": ">=2.0.0-internal.2.1.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/eslint-config-fluid": "^1.2.0",
-    "@fluidframework/fluid-static": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/mocha-test-setup": ">=2.0.0-internal.2.1.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/test-utils": "^1.3.0",
-    "@fluidframework/tinylicious-client": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
-    "@microsoft/api-extractor": "^7.22.2",
-    "@trivago/prettier-plugin-sort-imports": "^3.3.0",
-    "@types/chai": "^4.0.0",
-    "@types/mocha": "^9.1.1",
-    "chai": "^4.2.0",
-    "concurrently": "^6.2.0",
-    "copyfiles": "^2.4.1",
-    "eslint": "~8.6.0",
-    "eslint-config-prettier": "~8.5.0",
-    "eslint-plugin-chai-expect": "~3.0.0",
-    "mocha": "^10.0.0",
-    "nyc": "^15.0.0",
-    "prettier": "~2.6.2",
-    "rimraf": "^2.6.2",
-    "start-server-and-test": "^1.11.7",
-    "style-loader": "^1.0.0",
-    "tinylicious": "0.4.89251",
-    "typescript": "~4.5.5"
-  }
+	"name": "@fluid-tools/client-debugger",
```

Comment:
Looks like these are just formatting changes. We should probably revert them. Running `npm run format` in this directory should fix things up.

Reply:
Updated after `npm run format` Thx!